%time,field.error_time,field.error_type,field.error_source,field.error_text,field.fatal
1491588563716483923,1491588563.72,WIFI,beluga_alive.py,Lost connection with the user during teleoperation,1
1491588590220413234,1491588590.22,WIFI,beluga_alive.py,Lost connection with the user during teleoperation,1
1491588628124486200,1491588628.12,WIFI,beluga_alive.py,Lost connection with the user during teleoperation,1
